types/ovenplayer/tsconfig.json

```
@@ -0,0 +1,26 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6",
+            "dom"
+        ],
+        "target": "es5",
```

```suggestion
        "target": "es6",
```
that does not impact users code transpilation output, it's internal here for DT process. Keep this to default, unless there is a specific reason to change that value.

---

@peterblazejewicz  I got to know more about typescript. thank you.